docs/src/documentation/03-components/04-overlay/modal/03-accessibility.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: React
+redirect_from:
+  - /components/modal/accessibility/
+---
+
+# Accessibility
+
+## Modal
+
+The Modal component has been designed with accessibility in mind.
+
+To ease with keyboard navigation, when opening a modal, the focus is moved to the first focusable element inside the modal. It is also impossible to focus anything outside of the modal while it is open.
+When closing the modal, the focus can be moved back to the element that triggered the modal automatically, if the prop `triggerRef` is defined with a ref to that element.
+
+Besides that, assistive aria attributes are applied automatically to the modal and its children, to ensure that screen readers can announce the modal and its content correctly.
+
+If you prefer, you can also provide those attributes manually, as described on the table below:
+
+| Name            | Type     | Description                                                                                                                                                                 |
+| :-------------- | :------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ariaLabelledby  | `string` | The `id` of an element that serves as a label (title) for the modal.                                                                                                        |
+| ariaDescribedby | `string` | The `id` of an element that serves as a description for the modal.                                                                                                          |
+| ariaLabel       | `string` | Text that labels the modal content. Think of it as the title of the modal. This should be used if `title` is not passed to `ModalHeader` and `ariaLabelledby` is undefined. |
+
+All the props above are optional, but recommended to use to ensure the best experience for all users.
+
+If you use a ModalHeader with `title` and `description` props defined, they are automatically assigned as `aria-labelledby` and `aria-describedby`, respectively.
+However, if needed, you can overwrite the values by passing the corresponding props.
```

Comment:
I would mention that they need to be translated.

Reply:
Hmmm no. `ariaLabelledby` and `ariaDescribedby` receive IDs, so no need to translate them. The `title` and `description` props are visible on the screen, so it is not needed to advise that they need to be translated, as that's the rule for all visible text

Reply:
Well, yes, but you also mention that those elements can be hidden so they wouldn't be visible :D

Reply:
I get your point, and I admit we mentioned it in others, but I feel it deviates a bit from the real purpose of accessibility. But anyway, I'll add a line about it below